tests/_serialization/_data.py

```diff
@@ -15,7 +15,7 @@
 ]
 
 VALID_STRINGS = [
-    (properties.Property, 'hi', "<Property 'hi'>"),
+    (properties.Raw, 'hi', "<Property 'hi'>"),
```

Comment:
This works? Raw property probably isn't printing correctly, then.

Reply:
Can we either add a __str__ method to the Raw property or shift that into the base class?

Reply:
@andrewmillspaugh I think the issue is that the `Property` version is in the `__str__()` for `Property`, which is an abstract class.  So we can't create one and need to use a subclass that doesn't implement their own.  So if this is not what we want, maybe we should just remove the base method.